content/en/Platform Deep Dive/Collaboration/manage-notifications.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Manage Notifications"
+linkTitle: "Manage Notifications"
+weight: 20
+description: >
+  Explore Cobalt notifications.
+---
+
+{{% pageinfo %}}
+You can manage notifications for specific [pentests](#manage-pentest-notifications) and [findings](#manage-finding-notifications).
+{{% /pageinfo %}}
+
+By default, you get notifications for some activities related to your organization and pentests—both through email and in the Cobalt app. If you're not receiving notifications, make sure that you've confirmed your email address.
+
+{{%expand "View examples of activities." %}}
+<br>
+You get notified when someone has:
+
+- Created or deleted a pentest
+- Posted an update for a pentest
+- Reported a vulnerability
+- Changed a finding state
+- Invited users to the organization
+
+{{% /expand %}}
+<br>
+
+Your [user role](/platform-deep-dive/collaboration/organization/user-roles/) determines which notifications you get.
+
+## Manage Pentest Notifications
+
+When you collaborate on a pentest, you get notifications for all associated events.
+
+To manage your notification preferences for a pentest:
+
+1. On the pentest page, select **Settings** > **Notifications**.
+1. Select your preference:
+    - **Watching**: Receive notifications for all events, unless you've explicitly unsubscribed from a [finding thread](#manage-finding-notifications). This is the default option.
+    - **Participating**: Receive notifications when you're participating in a pentest, which includes:
+        - Someone mentioned you.
+        - You're an assignee on a finding.
+        - You're a commenter.
+    - **Ignoring**: Ignore all events, unless you've explicitly subscribed to a [finding thread](#manage-finding-notifications).<br><br>
+    ![Manage notification preferences for a pentest](/deepdive/ManagePentestNotifications.png "Manage notification preferences for a pentest")
+
+## Manage Finding Notifications
+
+You start getting notifications for a finding when:
+
+- You submitted the finding.
+- Someone assigned you to the finding.
+- You left a comment on the finding.
+- Someone mentioned you in a comment.
```

Comment:
I took these use cases from an internal Confluence page.

Reply:
> You start getting notifications for a finding when: You left a comment on the finding.

I wouldn't get a notification if I commented on a finding - however, if "someone left a comment on the Finding", you would receive a notification while "Watching" the pentest. 

Participating would mean you may only receive assigning and mentions.

Reply:
Also customers can't submit a finding, pentesters would create one.

Reply:
@1roshan Thank you, I addressed your feedback.

Reply:
Hi @1roshan, can we please finalize this pull request? I addressed your feedback and also changed the label for findings to **Follow**, based on the recent updates.